lib/krikri/harvester.rb

```diff
@@ -43,6 +43,12 @@ class << self
         def queue_name
           :harvest
         end
+        def entity_behavior
+          Krikri::OriginalRecordEntityBehavior
+        end
+        def generator_entity_behavior
+          raise 'A Harvester does not have a generator Activity.'
```

Comment:
This is a major code smell, to me.

See: https://github.com/dpla/KriKri/pull/116/files#r26081301